src/ansys/additive/core/microstructure.py

```diff
@@ -213,123 +230,130 @@ def material(self, value):
 
     @property
     def sample_min_x(self) -> float:
-        """Minimum x coordinate (m) of the geometry sample."""
+        """Minimum x coordinate of the geometry sample (m).
+
+        Valid values are from :obj:`MIN_POSITION_COORDINATE` to :obj:`MAX_POSITION_COORDINATE`.
```

Comment:
```suggestion
        Valid values are from the :obj:`MIN_POSITION_COORDINATE` value to
        the :obj:`MAX_POSITION_COORDINATE` value.
```
If I missed earlier formations of this, please revise. For clarity, a noun should follow a code entity.